src/main/java/frc/robot/commands/GotoNodes.java

```diff
@@ -142,6 +140,12 @@ private void changeTarget() {
             command.schedule();
     }
 
+    @Override
+    public void execute() {
+        if (nodePosition != nodePositionChooser.getSelected() || gridPosition != gridPositionChooser.getSelected())
+            changeTarget();
+    }
```

Comment:
this should be a private function (like all the code in execute pretty much)

Reply:
I don't see a reason to do that, if it was more cluttered I would agree